test/stdlogger.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { expect } = require('@hapi/code');
+const Lab = require('@hapi/lab');
+const Stdlogger = require('../lib/stdlogger');
+const { captureStd } = require('./helpers');
+
+const { describe, it } = exports.lab = Lab.script();
+
+
+describe('Stdlogger', () => {
+
+    it('logs info level to stdout', () => {
+
+        const std = captureStd();
+        const logger = new Stdlogger(std.stdout, std.stderr);
+        logger.info('my msg');
+        const result = std.complete();
+        expect(result.output).to.contain('my msg');
+    });
+
+    it('logs error level to stderr', () => {
+
+        const std = captureStd();
+        const logger = new Stdlogger(std.stdout, std.stderr);
+        logger.error('my error');
+        const result = std.complete();
+        expect(result.errorOutput).to.contain('my error');
+    });
+
+    it('formats additional fields on top level object', () => {
+
+        const std = captureStd();
+        const logger = new Stdlogger(std.stdout, std.stderr);
+        logger.info('my msg', null, { test: 'test' });
+        const result = std.complete();
+        expect(result.output).to.contain(',"test":"test"');
```

Comment:
Shouldn't we test for the exact string content? This test would not prevent the code from not displaying the event message and just the additionalFields IMO.

Reply:
Updated the tests... don't want to test for exact structure as a timestamp is added, plus the ordering can change later and don't want the tests to be too tied to specific ordering.